persistence/postgre.py

```diff
@@ -0,0 +1,170 @@
+from sqlalchemy.orm import Session
+from sqlalchemy import text
+from models.courses import Courses
+from models.colaborators import Colaborators
+from models.enrolled import Enrolled
+
+
+class DB:
+    def __init__(self, engine):
+        self.session = Session(engine)
+
+    def addCourse(self, courseInfo):
+        name = courseInfo["name"]
+        exams = courseInfo["exams"]
+        creator_id = courseInfo["user_id"]
+        type = courseInfo["type"]
+        subscription = courseInfo["subscription"]
+        location = courseInfo["location"]
+        description = courseInfo["description"]
+        hashtags = courseInfo.get("hashtags", "")
+        c = Courses(
+            name=name,
+            exams=exams,
+            creator_id=creator_id,
+            type=type,
+            subscription=subscription,
+            description=description,
+            hashtags=hashtags,
+            location=location,
+            cancelled=0,
+        )
+        self.session.add(c)
+        self.session.commit()
+
+    def getCourse(self, courseId):
+        course = self.session.query(Courses).get(courseId)
+        if not course:
+            return None
+        dic_course = {
+            "name": course.name,
+            "exams": course.exams,
+            "creator_id": course.creator_id,
+            "type": course.type,
+            "subscription": course.subscription,
+            "description": course.description,
+            "hashtags": course.hashtags,
+            "location": course.location,
+            "cancelled": course.cancelled,
+        }
+        return dic_course
+
+    def getCourses(self, courseFilters):
+        offset = courseFilters["offset"]
+        limit = courseFilters["limit"]
+        where_clause = "WHERE cancelled = 0"
```

Comment:
En un futuro hay que hacer que el cancelled sea una variable mas, no siempre vamos a querer usar este metodo con cancelled = 0, ademas cancelled es un tiny int, por lo que un WHERE not cancelled ... funciona